### PR TITLE
Add ARDN hackathon blog post

### PR DESCRIPTION
**Pull request by KristinaRiemer**

### CODE REVIEW

File: content/post/2020-05-08-ardn-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - Brief summary of other datasets
+  - ISU
+  - MSU
+  - UGA
+
+- paragraph 4
+  - Summary of our part: putting TR data from BrAPI into ARDN, so any BrAPI-compliant datasets can be automated into that format
+
+- paragraph 5
+  - Expanding to even more datasets, through Vmapper for CSV and for EML (DataONE datasets compatible)
```
Comment (dlebauer) [inline content/post/2020-05-08-ardn-hackathon.md]: and our vision - there is definitely not a single one-size-fits-all data standard, but when communities coalesce around a few, it is easier to build tools to make these interoperate, and thus harmonize data.

**Comment by dlebauer** [comment]
Looks great! Here are some slides that can be exported as figures and used in this post https://emailarizona-my.sharepoint.com/:p:/g/personal/dlebauer_email_arizona_edu/EV-Jbh_rj_5Gg6-4_5EuxAQBv4a7TOn_pF-5Z4mQ5w_jxg?e=A2FQMb

thumbnail: 

image

**Comment by KristinaRiemer** [comment]
There is now a very rough written draft of the blog post. It definitely needs to be polished, and maybe some content needs to be added/removed. It also needs a lot more links to people, datasets, etc.!

**Comment by KristinaRiemer** [comment]
This looks great @dlebauer, thanks for improving the language and terminology used. I fixed a few small errors. I'll send this to Cheryl for feedback.

**Comment by KristinaRiemer** [comment]
@dlebauer added Cheryl's comments. Let me know if this is ready to go on the (old) website.

**Comment by KristinaRiemer** [comment]
This [has been published on the old site](https://datascience.cals.arizona.edu/news/2020/06/hackathon-building-agricultural-research-data-network), but I never did so here. Merging now.

**Pull request merged**